### ISSUE TITLE
135 => 136 update: Programmable controllers randomly resetting

### ISSUE THREAD

**Issue by Axylxys** [MineMaarten/PneumaticCraft#714]
I'm not sure it's intended, but going from version 135 to 136, i found my Programmable Controllers still there, however they lost all upgrades and drones/network API that were inside, including their programs.

That makes me a bit worried, as such a wipe on the Programmable Controllers could easily screw up a whole base once i get more drones running like that, as i plan them to handle quite a lot of stuff. Thankfully i only had two of them there.

**Comment by Axylxys** [comment]
I edited the title, it actually has nothing to do with updating it seems. The controllers just randomly reset (losing all upgrades, the network API/drone and the program puzzle pieces used on that API/drone). I noticed that after looking hopelessly for my API on my builder drone, then i noticed the upgrades on the controller were missing too. After 5mn of running around thinking i was turning crazy, i went to look at the other two programmable controllers: no upgrades, no API or anything inside.

All are chunk loaded of course, i'm not sure if it happens on server reboot or what could be triggering that. They seemed to work fine otherwise, they're just not really usable for anything sustained right now as they can vanish in thin air anytime.
